package.json

```diff
@@ -32,6 +32,7 @@
     "dbus": "^1.0.7",
     "mdns-resolver": "^1.0.0",
     "semver": "^7.3.2",
+    "sinon": "^11.1.2",
```

Comment:
Do we need to update @types/sinon too? It's currently still at `^10.0.0`

Reply:
latest @types/sinon is v10.0.11 so I can't upgrade to 11.1.2 unfortunately.

I had to check what the convention was for definitely typed package versions which https://github.com/DefinitelyTyped/DefinitelyTyped/issues/13225 confirmed that they should match the version of the package it is adding types for.